### DISCUSSION TITLE
Could the close button at top-right corner of plotter window be disabled ?

### DISCUSSION THREAD

**Discussion by ZiguoAtGitHub** [marcomusy/vedo#886] [Q&A] [answered]
Hi Marco,
Is there some way to disable or hide the close button at top-right corner of plotter window?
Thanks!!
![image](https://github.com/marcomusy/vedo/assets/116484589/6e5c8197-911f-4bf0-aa19-dd462ed1513e)

**Comment by marcomusy** [comment]
Hi Ziguo, you may try
```
plt.window.BordersOff()
```
but in some cases this might not work or cause crashes..  see [here](https://vtk.org/doc/nightly/html/classvtkRenderWindow.html#a58659608d2dced1fd87b151473b43840).

**Reply by ZiguoAtGitHub** [reply]
Hi Marco,
First, thanks for your  quick response.
I tried it does work but the minimize button and maximize button are also disappeared....

**Reply by marcomusy** [reply]
You can only get rid of the window as a whole but cannot access the internals.. that has nothing to do with vedo.. it's the operative system.

**Reply by ZiguoAtGitHub** [reply, marked as answer]
Thanks for your explanations and I got the root of  the problem.
I found an alternative way to get what I want: Re-produce plotter window when needed.😃
